Sources/Fullscreen/Settings/SettingsView.swift

```diff
@@ -8,8 +8,9 @@ import FinniversKit
 // MARK: - View Model
 
 public protocol SettingsViewModel: ObservableObject {
+    var viewTitle: String? { get }
     var sections: [SettingsSection] { get }
-    var versionText: String { get }
+    var versionText: String? { get }
```

Comment:
Just a heads up: this is a breaking change 🙂

Reply:
Agree